lib/christmas_tree_formatter.rb

```diff
@@ -1,40 +1,40 @@
 # encoding: UTF-8
 
-require "rspec/core"
-require "rspec/core/formatters/base_text_formatter"
+if defined? RSpec
```

Comment:
I think either way is fine, actually! Since the `.rspec` file probably won't load until the `RSpec` module is initialized, we don't run any risk of this file being required before RSpec is.